<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Root_Left_Left">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Root_Right_Left">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>160</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Root_Right_Right">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>160</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Root_Right">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Root">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BASA</string>
    </property>
   </widget>
   <widget class="QLabel" name="Root_Left_Right">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Root_Left">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter key/value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineKey">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineValue">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Remove">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Find">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Find">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
